0011-copy_python_exe.md

```diff
@@ -0,0 +1,37 @@
+- Feature Name: copy_python_exe
+- Start Date: 2019-06-12
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+For maintenance and bug-reporting, it would be beneficial if the salt client processes in the Windows task manager would be named `salt-minion.exe` instead of `python.exe`
+
+# Motivation
+[motivation]: #motivation
+
+For maintenance and bug-reporting.
+
+# Design
+[design]: #detailed-design
+
+copy python.exe to salt-minion.exe and reference the latter, e.g in salt-minion.bat
```

Comment:
There may some better alternatives than strictly copying python.exe, like using the Setuptools approach. I think that the SEP should just specify that we want to create `salt-*.exe`, and provide a couple of options, since it's likely that there will be issues with one approach or another.

Reply:
I no longer want or can discuss the implementation. Please feel free to change the detailled design.

Reply:
I will try to find time for a test